samples/analyze.v1p2beta1.js

```diff
@@ -0,0 +1,383 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the `License`);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an `AS IS` BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+function analyzeTextGCS(gcsUri) {
```

Comment:
We are starting to change all of our code samples over to use `async/await` syntax.  Let's just make this an async function.

Reply:
For sure. Have changed the code. Please have a look and let me know if it all looks ok.